text/2020-09-03-factored-prefix.md

```diff
@@ -0,0 +1,71 @@
+# Factored Key Prefix
+
+## Summary
+
+TiKV provides a facility where the client can set a key prefix that will be used for all keys in its requests.
+
+## Motivation
```

Comment:
The motivation is not clear to me. Is it for multi-tenancy? What's the higher level use case look like with multiple prefixes?